packages/location-field/src/stories/Mobile.story.js

```diff
@@ -129,6 +130,7 @@ export const Styled = () => (
   <StyledLocationField
     currentPosition={currentPosition}
     geocoderConfig={geocoderConfig}
+    layerColorMap={layerColorMap}
```

Comment:
Colors don't seem to take effect in any mobile stories, is that intended? (and if not, then why is the passing of the prop needed?)

Reply:
I'm seeing colors in the mobile stories (when making a search). Are these not appearing for you?

Reply:
Okay, it's for when making a search.